src/layer/XYZ.ts

```diff
@@ -54,7 +54,7 @@ type XYZEventsType = EventsHandler<XYZEventsList> & EventsHandler<LayerEventsLis
  *     attribution: 'Data @ <a href="http://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="http://www.openstreetmap.org/copyright">ODbL</a>'
  * });
  */
-class XYZ extends Layer {
+export class XYZ extends Layer {
```

Comment:
Frankly, I prefer export {XYZ}, but okay, I suppose you have a strong argument about this:)